### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by oyelajakenny**
I have implemented the best Performing currency pair and market status functionalities.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,11 +9,19 @@
       src="https://kit.fontawesome.com/0f4d71be75.js"
       crossorigin="anonymous"
     ></script>
+   
   </head>
   <body>
+    <section class="top-bar">
+<div class="top-bar-wrapper">
```
Comment (vdaguenet) [inline index.html]: Don't forget to indent the HTML properly.

File: app.js [hunk 2]
```diff
@@ -135,10 +138,31 @@ document.addEventListener("DOMContentLoaded", () => {
     }
   });
 
+  //Best performing currency pair
+
+  function topCurrency() {
```
Comment (vdaguenet) [inline app.js:143]: Nice addition!

**Review by vdaguenet — APPROVED** [review]
Good job 👍🏻

File: index.html (hunk 1 above)
Comment (oyelajakenny) [reply index.html]: Thank you for your feedback @vdaguenet, I have done that and will put that in mind moving forward

**Pull request merged**